cpp/ql/src/semmle/code/cpp/dataflow/EscapesTree.qll

```diff
@@ -4,6 +4,14 @@
  * passed to a function, or similar.
  */
 
+/*
+ * Maintainer note: this file is one of several files that are similar but not
+ * identical. Many changes to this file will also apply to the others:
+ * - AddressConstantExpression.qll
+ * - AddressFlow.qll
+ * - EscapesTree.qll
+ */
```

Comment:
I think it's possible for these three files to share code, but I don't think it would make this PR any easier to review.